helm/basex-validator/templates/_helpers.tpl

```diff
@@ -0,0 +1,32 @@
+{{/* vim: set filetype=mustache: */}}
```

Comment:
this file is generated boilerplate, I've simplified it in https://github.com/libero/publisher-chart/blob/master/helm/publisher/templates/_helpers.tpl#L1-L6:
- leave only `basex-validator.chart`
- inline `.Release.Name` wherever `.fullname` is used
- inline `.Chart.Name` wherever `.name` is used